catalog/app/components/Markdown/Markdown.tsx

```diff
@@ -189,7 +189,7 @@ const linkHandler =
       const rel = nofollow ? ' rel="nofollow"' : ''
       return `<a href="${(Remarkable as unknown as RemarkableWithUtils).utils.escapeHtml(
```

Comment:
let's adjust the formatting here a little bit: move `(Remarkable as unknown as RemarkableWithUtils).utils.escapeHtml(...` to a separate line and assign it to a variable for readability (i know it's kinda out of scope, but anyways, let's "fix" it since we're touching it)